src/Foundatio.RabbitMQ/Messaging/RabbitMQMessageBus.cs

```diff
@@ -32,12 +33,11 @@ public RabbitMQMessageBus(RabbitMQMessageBusOptions options) : base(options)
         // in case the server is restarted or there has been any network failures
         // Topology ( queues, exchanges, bindings and consumers) recovery "TopologyRecoveryEnabled" is already enabled
         // by default so no need to initialize it. NetworkRecoveryInterval is also by default set to 5 seconds.
-        // it can always be fine tuned if needed.
+        // it can always be fine-tuned if needed.
         _factory = new ConnectionFactory
         {
             Uri = new Uri(options.ConnectionString),
-            AutomaticRecoveryEnabled = true,
-            DispatchConsumersAsync = true
```

Comment:
This was removed and added to IConnection but looking through all the rabbit code I don't see any usages or how to set this.